packages/@webex/plugin-encryption/README.md

```
@@ -0,0 +1,78 @@
+      const decryptedFileBuf = await webex.cypher.downloadAndDecryptFile(attachmentURL);
+      // Do something with the decrypted file buffer
+    } catch (error) {
+      // Handle error
+    }
+  }).catch((err) => {
+    // Handle error
+  });
+});
+```
+
+#### Development
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)